src/main/java/com/softserve/actent/model/entity/Review.java

```diff
@@ -5,12 +5,7 @@
 import lombok.NoArgsConstructor;
 import lombok.NonNull;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.Table;
```

Comment:
Replace with single imports.

Reply:
done

Reply:
Still asterisk instead of single imports.